app/views/layouts/_header.html.erb

```diff
@@ -35,14 +35,14 @@
           <a class="btn" href="https://donorbox.org/operationcode"
             data-constrainwidth="false" data-beloworigin="true" data-activates="Donate">Donate</a>
         </li>
-        
+
         <% if veteran_signed_in? %>
           <li><%= link_to 'Profile', profile_path %></li>
           <li><%= link_to 'Sign Out', destroy_veteran_session_path, method: :delete %></li>
         <% else %>
           <li>
             <% ab_test(:signup_button_size, 'btn', '') do |button_size| %>
-            <%= link_to 'Join', new_veteran_registration_path, class: button_size, style: 'background-color: #B23900;' %>
+            <%= link_to 'Join', new_veteran_registration_path, class: 'btn', style: 'background-color: #26A69A;' %>
```

Comment:
This will break the AB test. `button_size` is passed into the block from the last two arguments to the `ab_test` method.

I also don't think we want to change the background color but the text color.